app/src/main/java/com/blackboxteam/app/blackbox/util/UserSessionPersistence.java

```diff
@@ -0,0 +1,38 @@
+package com.blackboxteam.app.blackbox.util;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.widget.Toast;
+
+/**
+ * UserSessionPersistence
+ *
+ * Manages user session
+ */
+public class UserSessionPersistence {
+    private Context mContext;
+    private String user;
+    private SharedPreferences sharedPreferences;
+
+    public UserSessionPersistence(Context context) {
+        mContext = context;
+    }
+
+    // Save the user login/signIp info
+    public boolean saveUserInfo(String email, String password) {
+        sharedPreferences = mContext.getSharedPreferences("userInfo", Context.MODE_PRIVATE);
+
+        SharedPreferences.Editor editor = sharedPreferences.edit();
+        editor.putString("email", email);
+        editor.putString("password", password);
```

Comment:
In practice, shared preferences keys are defined as static local constants at the top of the class. We define a static local constants as follows:
``` private static final String USER_EMAIL = "USER_EMAILE";```

So instead of passing a string value each time you want to save or get the user email, we simply pass in this local constant.

Reply:
Done